lib/meta-boxes.php

```diff
@@ -300,5 +300,44 @@ function igv_cmb_metaboxes() {
     // 'timezone_meta_key' => $prefix . 'timezone', // Optionally make this field honor the timezone selected in the select_timezone specified above
   ) );
 
+// COMMITTEE
+  
+  $committee_metabox = new_cmb2_box( array(
+    'id'            => $prefix . 'committee_metabox',
+    'title'         => __( 'Committee Members', 'cmb2' ),
+    'object_types'  => array( 'committee', ), // Post type
+  ) );
+
+  $committee_members = $committee_metabox->add_field( array(
+    'id'          => $prefix . 'committee_members',
+    'type'        => 'group',
+    'description' => __( '', 'cmb2' ),
+    'options'     => array(
+      'group_title'   => __( 'Member {#}', 'cmb2' ), // {#} gets replaced by row number
+      'add_button'    => __( 'Add Another Member', 'cmb2' ),
+      'remove_button' => __( 'Remove Member', 'cmb2' ),
+      'sortable'      => true, // beta
+      // 'closed'     => true, // true to have the groups closed by default
+    ),
+  ) );
+
+  $committee_metabox->add_group_field( $committee_members, array(
+    'name'        => __( 'Name', 'cmb2' ),
+    'description' => __( '', 'cmb2' ),
+    'id'          => 'name',
+    'type'        => 'text_medium',
+  ) );
+
+  $committee_metabox->add_group_field( $committee_members, array(
+    'name'    => __( 'Bio', 'cmb2' ),
+    'id'      => 'bio',
+    'type'    => 'wysiwyg',
```

Comment:
This has the repeating group wysiwyg issue that is still unfixed in CMB. Does this need to be wysiwyg? If not thats the simple fix.